qf-polkavm-sdk/src/bump_allocator.rs

```diff
@@ -0,0 +1,123 @@
+use core::alloc::{GlobalAlloc, Layout};
+
+static mut INNER: Option<InnerAlloc> = None;
+
+/// Memory heap
+static mut HEAP: [u8; 1024 * 1024] = [0; 1024 * 1024];
```

Comment:
Is this static memory will be allocated even if we nave no "global-allocator" feature enabled?